samples/bluetooth/rssi_nrf53/src/main.c

```diff
@@ -0,0 +1,130 @@
+/* main.c - Application main entry point */
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <sys/printk.h>
+#include <sys/util.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/hci_vs.h>
+
+struct rssi_wq_data {
+	struct k_work work;
+	bt_addr_le_t addr;
+	int8_t rssi;
+	uint8_t adv_type;
+} rssi_wq_data;
+
+static int8_t get_chip_temp(void)
```

Comment:
I believe the preferred way to obtain the die temperature is by using the sensor API (`SENSOR_CHAN_DIE_TEMP`). That sensor implementation will call the mpsl API obtaining the die temperature.

Reply:
@anangl , could you comment on the preferred method to obtain the sensor value in this case?

Reply:
yup, that's why I have implemented this hci command, else I would just have used the sensor API